common/filestore_utils.py

```diff
@@ -39,8 +39,8 @@ def get_impl():
 
 def cp(source, destination, recursive=False, parallel=False, expect_zero=True):  # pylint: disable=invalid-name
     """Copies |source| to |destination|. If |expect_zero| is True then it can
-    raise subprocess.CalledProcessError. |parallel| is only effective for
-    gsutil."""
+    raise subprocess.CalledProcessError. |parallel| is only used when by the
```

Comment:
Shoot, i had a typo in my comments to you on the last PR.
Here is the correct version:
`|parallel| is only used by the gsutil implementation.`

Sorry!